### PR TITLE
Release blog post

### PR DESCRIPTION
**Pull request by christian-bromann**
First draft of the release blog post.

### CODE REVIEW

File: website/blog/2018-12-19-webdriverio-v5-released.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+<br>
+As part of that effort we've renamed a lot of commands to align them to the pattern that has been used in the protocol. The base WebDriver client also now returns the `value` property of all protocol commands responses so that we were able to get rid of a lot of duplication (e.g. `title` and `getTitle` is now called `getCurrentTitle`).<br>
+<br>
+Additionally we have seen a lot of confusion around the WebdriverIO API and the fact that it sometimes takes selectors as first argument and sometimes not which made it really hard to e.g. build a consistent and useful TypeScript definition for it. In the new version __we got rid of selectors as command parameters__ and defined a set of commands only callable when using the browser/client instance or when using an element instance. In order to click on an element you now need to first fetch it to receive an element object and then call the click command on that instance, e.g.
+
+```js
+browser.click('#elem') // throws 'browser.click is not a function'
+
+const elem = $('#elem')
+elem.click() // clicks successfully on the element
```
Comment (pmerwin) [inline website/blog/2018-12-19-webdriverio-v5-released.md]: $('#elem').click() -> is this , also, possible?

File: website/blog/2018-12-19-webdriverio-v5-released.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+We are still in the process of finalising this effort as we want to continue providing you a way to simply [change the version](https://github.com/webdriverio/webdriverio/issues/3147) of the docs. We are also looking into [providing multiple translation](https://github.com/webdriverio/webdriverio/issues/3148) of the docs so that people who don't speak english can also understand and use WebdriverIO. Please reach out to us on [Twitter](https://twitter.com/webdriverio) or directly on the issue thread if you want to help us out here. It is probably one of the best ways to get involved into an open source project.
+
+## How To Upgrade To v5
+
+There are much much more things that we have been working on over the last month that you might like to find out. Check out the official [changelog](https://github.com/webdriverio/webdriverio/blob/master/CHANGELOG.md#v500-2018-12-20) to find all changes that describe the new version. We might will update it over time since we couldn't keep a list of every detail that might have change. We would also like to ask you to have a look into the new [guide section](https://webdriver.io/docs/gettingstarted.html) and our updated docs in general.<br>
```
Comment (pmerwin) [inline website/blog/2018-12-19-webdriverio-v5-released.md]: There are much much - > There are many?

File: website/blog/2018-12-19-webdriverio-v5-released.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+- try to run that first test suite you modified `$ ./node_modules/.bin/wdio wdio.config.js --spec ./path/to/modified/test.js`
+- take on the next test file and work your way through them
+- add reporters and services back into your `wdio.conf.js` and see if they work as usual (__Note:__ it is possible that services or reporters that you have used aren't ported to v5 yet, if so please raise an issue in the repository if that community package or try to port it)
+
+<br>Again, reach out in the support channel if you have issues porting your test suite. Chances are that we have missed to port a functionality that you have been using in your test. Thanks to the new project structure we can quickly fix this and provide an update version for you!<br>
+<br>
+We will release further blog article in our new blog with tutorials on how to upgrade WebdriverIO to v5 soon. You can also checkout the excellent [video series](https://www.youtube.com/watch?v=MO8xeC-w2Og&list=PL8HowI-L-3_9Ep7lxVrRDF-az5ku4sur_) from our beloved [Will Brock](https://twitter.com/willbrock) on the new release. An update to the [WebdriverIO Learning Course](https://learn.webdriver.io/) is also already in work.
```
Comment (pmerwin) [inline website/blog/2018-12-19-webdriverio-v5-released.md]: Awesome!!!

File: website/blog/2018-12-19-webdriverio-v5-released.md (hunk 1 above)
Comment (christian-bromann) [reply website/blog/2018-12-19-webdriverio-v5-released.md]: yes

**Comment by christian-bromann** [comment]
Reviewed by @abjerstedt in #3150

**Pull request merged**